File: Service/Models/API/RoundInfo.cs
namespace BetSnooker.Models.API
{
    public class RoundInfo
    {
        public int Round { get; set; }

        public string RoundName { get; set; }

        public int EventId { get; set; }

        public int Distance { get; set; }

        public int NumMatches { get; set; }
    }
}

File: Web.UI_old/src/app/_models/roundInfo.ts
export class RoundInfo {
    round: number;
    roundName: string;
    eventId: number;
    distance: number;
    numMatches: number;
    actualStartDate: Date;
    started: boolean;
    finished: boolean;
    isFinalRound: boolean;

    startDate: string; // this is only to format actualStartDate
}
